lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,138 +1,232 @@
+                    <button *ngIf="canEditGeneralInfo"
+                            mat-icon-button
+                            (click)="onToggleGeneralInfoEdit($event)"
+                            [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
+                            [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
+                            data-automation-id="meta-data-general-info-edit"
+                            class="adf-edit-icon-buttons">
+                        <mat-icon>mode_edit</mat-icon>
+                    </button>
+                    <div *ngIf="isEditingGeneralInfo" class="adf-metadata-action-buttons">
+                        <button mat-icon-button
+                                [attr.title]="'CORE.METADATA.ACTIONS.CANCEL' | translate"
+                                (click)="onCancelGeneralInfoEdit($event)"
+                                data-automation-id="reset-metadata"
+                                class="adf-metadata-action-buttons-clear">
+                            <mat-icon>clear</mat-icon>
+                        </button>
+                        <button mat-icon-button
+                                [attr.title]="'CORE.METADATA.ACTIONS.SAVE' | translate"
+                                (click)="onSaveGeneralInfoChanges($event)"
+                                color="primary"
+                                data-automation-id="save-general-info-metadata"
+                                [disabled]="!hasMetadataChanged">
+                            <mat-icon>check</mat-icon>
+                        </button>
+                    </div>
+                </adf-content-metadata-header>
             </mat-expansion-panel-header>
             <adf-card-view
                 (keydown)="keyDown($event)"
                 [properties]="basicProperties$ | async"
-                [editable]="editable"
+                [editable]="isEditingGeneralInfo"
                 [displayEmpty]="displayEmpty"
                 [copyToClipboardAction]="copyToClipboardAction"
                 [useChipsForMultiValueProperty]="useChipsForMultiValueProperty"
                 [multiValueSeparator]="multiValueSeparator">
             </adf-card-view>
         </mat-expansion-panel>
+
         <ng-container *ngIf="displayTags">
-            <mat-expansion-panel *ngIf="!editable">
+            <mat-expansion-panel
+                (opened)="isTagPanelExpanded = true"
+                (closed)="isTagPanelExpanded = false"
+                hideToggle
+                [expanded]="isTagPanelExpanded">
                 <mat-expansion-panel-header>
-                    <mat-panel-title>{{ 'METADATA.BASIC.TAGS' | translate }}</mat-panel-title>
+                    <adf-content-metadata-header [title]="'METADATA.BASIC.TAGS'" [expanded]="isTagPanelExpanded">
+                        <div class="adf-tags-buttons">
```

Comment:
Is that div needed? It contains only single button so class can be applied just to button (especially that it has no extra styles, just reference in tests)

Reply:
removed the div, also discovered couple of useless unit tests to cleanup